opensuse-kubectl-docker/Makefile

```diff
@@ -0,0 +1,8 @@
+IMAGE ?= splatform/opensuse-kubectl
+TAG ?= latest
+
+build:
+	docker build -t ${IMAGE}:${TAG} - < Dockerfile
+
+push: build
+	docker push ${IMAGE}:${TAG}
```

Comment:
You should configure whatever editor you're using to force end of line at the end of files so git doesn't complain.